src/Acuminator/Acuminator.Benchmark/RoslynHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Configuration;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Acuminator.Analyzers.StaticAnalysis;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.MSBuild;
+
+namespace Acuminator.Benchmark
+{
+	public static class RoslynHelper
+	{
+		public static ImmutableArray<DiagnosticAnalyzer> GetAvailableAnalyzers()
+		{
+			return Assembly.GetAssembly(typeof(PXDiagnosticAnalyzer)).ExportedTypes
+				.Where(t => !t.IsAbstract && typeof(DiagnosticAnalyzer).IsAssignableFrom(t))
+				.Select(Activator.CreateInstance)
+				.Cast<DiagnosticAnalyzer>()
```

Comment:
@dropsonic this way we analyze only the perfomance of the aggregators. Shouldn't we add more granulated benchmarks too?

Reply:
We analyze performance for all analyzers, not for the aggregators.
Detailed telemetry is available in a separate project called "Acuminator.PerformanceReport".
But all these benchmarks are kinda useless because Visual Studio handles multithreading in its own very special way.

Reply:
@dropsonic but if I remeber correctly, only the aggregators derive from `PXDiagnosticAnalyzer`. 
Aggregated analyzers for PXGraph for example just implement `IPXGraphAnalyzer` interface. I haven't found the usage of this interface. Do you measure perfomance for such aggregated analyzers?

Reply:
I got it. No, by now there is no such need.

Reply:
ok